client/src/components/DisplayTable/DisplayTable.js

```diff
@@ -113,15 +113,15 @@ export class DisplayTable extends React.Component {
         },
       }
       */,
+      page_size_increment,
       util_components,
-      size_to_paginate,
```

Comment:
Bit of an unclear purpose, could only see it as being, in practice, used to disable pagination (in which case it could have just been a `disable_pagination` prop).

Reply:
... I didn't add one, but an explicit `disable_pagination` would still be useful, probably.